digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/client/RoadLinkChangeClient.scala

```diff
@@ -138,8 +137,7 @@ class RoadLinkChangeClient {
       }
     }
 
-    val request = ListObjectsV2Request.builder().bucket(s3Bucket).build()
-    val objects = s3Client.listObjectsV2(request).contents().asScala.toList
+    val objects = s3Service.listObjects(s3Bucket).asScala.toList
```

Comment:
ei varmaan tuota ongelmia nyt mut eikös tää nyt hae kaikki items jotka on s3?
Vähän mitäs tapahtuu sitten kun meillä on viikon tai parin verran tavaraa. 
Tää operaatio saattaa muurtua aika raskaaksi.

Reply:
@reetta-kivisto Kuis pitkään me säilytetään muutoksia S3 ?

Reply:
Printti näyttäis et tulee vain metatiedot (nimi, koko jne.). Pitää varmaan keksiä jotain muuta, jos menee raskaaksi.

Reply:
Ei olla vielä määritetty sille lifecycle policyä. Pitäisi kyllä lisätä joku aika